src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -275,10 +277,12 @@ public void spotCheck2() throws IOException {
 
         final ArgumentsBuilder arguments = new ArgumentsBuilder();
 
+        arguments.addArgument("verbosity", "INFO");
+
         arguments.addVCF(new File(inputFile));
         arguments.addReference(new File("/Users/jonn/Development/references/Homo_sapiens_assembly19.fasta"));
 
-        arguments.addArgument(FuncotatorArgumentDefinitions.DATA_SOURCES_PATH_LONG_NAME, "/Users/jonn/Development/funcotator_dataSources.v1.1.20180208");
+        arguments.addArgument(FuncotatorArgumentDefinitions.DATA_SOURCES_PATH_LONG_NAME, "/Users/jonn/Development/funcotator_dataSources_latestjnkjjk");
```

Comment:
This pathname can't possibly be correct.  Must be something that would be on anyone's dev machine.

Reply:
This test is intermediate and planned to be phased out.

It is checked in so that I can run my local test data and not have to worry about rewriting tests every time I need to debug something.

I added the flag `doDebugTests` to control for this and a unit test to ensure that `doDebugTests` is always set to false for the tests to pass.

I want to leave these in for now so I can continue to test locally with different settings easily.